pkgs/servers/web-apps/engelsystem/default.nix

```diff
@@ -0,0 +1,61 @@
+{ stdenv
+, fetchzip
+, php
+, phpExtensions
+, makeWrapper
+, writeText
+}:
+
+
+let
+  extraInit = writeText "costum-php.ini" ''
+    extension=${phpExtensions.json}/lib/php/extensions/json.so
+    extension=${phpExtensions.filter}/lib/php/extensions/filter.so
+    extension=${phpExtensions.mysqlnd}/lib/php/extensions/mysqlnd.so
+    extension=${phpExtensions.mysqli}/lib/php/extensions/mysqli.so
+    extension=${phpExtensions.pdo}/lib/php/extensions/pdo.so
+    extension=${phpExtensions.pdo_mysql}/lib/php/extensions/pdo_mysql.so
+  '';
```

Comment:
There are two functions which do this automatically for you - `php.withExtensions` and `php.buildEnv`; please use them instead. You can find the documentation in `doc/languages-frameworks/php.section.md`.

Reply:
Using this you shouldn't need to wrap `migrate` below with the env variable `PHP_INI_SCAN_DIR` because it will interfere with the wrapper.

Reply:
Didn't saw the functions, I'm trying to build it. but I can't get it to work with the patchShebang function. Will commit my current work, and maybe someon can point me out why it is not working with patchShebangAuto

Reply:
No problem, it's a very recent development :) It doesn't work because of a bug in the version of nixpkgs you're using. If you rebase, it should work.